src/handlers/HeyHandler.test.ts

```diff
@@ -13,7 +13,7 @@ const DefaultGroupMeInfo = {
   source_guid: "GUID",
   system: false,
   text: "Hello world ☃☃",
-  user_id: "1234567890"
+  user_id: "1234567890",
```

Comment:
how i fix this ` 16:3  error  Identifier 'user_id' is not in camel case      @typescript-eslint/camelcase`